drag_and_drop_v2/public/js/drag_and_drop.js

```diff
@@ -320,28 +320,30 @@ function DragAndDropTemplates(configuration) {
     var itemFeedbackPopupTemplate = function(ctx) {
         var popupSelector = 'div.popup';
         var msgs = ctx.feedback_messages || [];
+        var have_messages = msgs.length > 0;
         var popup_content;
 
         if (msgs.length > 0 && !ctx.last_action_correct) {
             popupSelector += '.popup-incorrect';
         }
 
-        if (msgs.length > 1) {
-            popup_content = h("div.popup-content", {}, [
+        if (ctx.mode == DragAndDropBlock.ASSESSMENT_MODE) {
+            var content_items = [
```

Comment:
@e-kolpakov `message_items` would be a little more to the point I think.

Thanks for making the behavior consistent!